### PR TITLE
add process for querying WMO RA by geometry

### PR DESCRIPTION
**Pull request by tomkralidis**
This PR supports wis2box metadata UI/UX to be able to derive the WMO RA from a (WKT) geometry.  Accepts any valid WKT as input, and will return any RAs that intersect with same.

### CODE REVIEW

File: wis2box_api/plugins/process/wmo_ra.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        'result': {
+            'title': 'WMO RA result',
+            'description': 'WMO RA result',
+            'schema': {
+                'type': 'object',
+                'contentMediaType': 'application/json'
+            }
+        }
+    },
+    'example': {
```
Comment (david-i-berry) [inline wis2box_api/plugins/process/wmo_ra.py:71]: Example output?

File: wis2box_api/plugins/process/wmo_ra.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        if geometry is None:
+            msg = 'Missing WKT geometry'
+            LOGGER.error(msg)
+            raise ProcessorExecuteError(msg)
+
+        driver = ogr.GetDriverByName('GeoJSON')
+        dataSource = driver.Open(wmo_ra_geojson, 0)
+        layer = dataSource.GetLayer()
+
+        geometry = ogr.CreateGeometryFromWkt(data['geometry'])
```
Comment (david-i-berry) [inline wis2box_api/plugins/process/wmo_ra.py:119]: What if mis-formed WKT? do we want to catch that here or is None returned (and caught below)?

File: wis2box_api/plugins/process/wmo_ra.py (hunk 1 above)
Comment (tomkralidis) [reply wis2box_api/plugins/process/wmo_ra.py:71]: Examples are generated from the OpenAPI schema.  In the 200 case, pygeoapi keeps this generic by not defining a schema per se.

File: wis2box_api/plugins/process/wmo_ra.py (hunk 2 above)
Comment (tomkralidis) [reply wis2box_api/plugins/process/wmo_ra.py:119]: `ogr.CreateGeometryFromWkt` will return `None` for invalid WKT, and an exception is thrown thereafter.

**Review by david-i-berry — APPROVED** [review]

**Pull request merged**